app/src/main/java/ch/epfl/skysync/database/DateUtility.kt

```diff
@@ -57,4 +67,40 @@ object DateUtility {
     val formatter = DateTimeFormatter.ofPattern("HH:mm:ss")
     return LocalTime.parse(raw, formatter)
   }
+
+  /**
+   * Converts a [Date] object to a [LocalTime] object.
+   *
+   * @param date The [Date] object to be converted.
+   * @return The equivalent [LocalTime] object.
+   */
+  fun dateToLocalTime(date: Date): LocalTime {
+    val instant = date.toInstant()
+    val zonedDateTime = instant.atZone(ZoneId.systemDefault())
+    return zonedDateTime.toLocalTime()
+  }
+
+  /**
+   * Converts a [LocalDate] object to a string.
+   *
+   * @param date The [LocalDate] object to be converted.
+   * @return The equivalent [LocalTime] object or --/--/-- if null.
+   */
+  fun localDateToString(date: LocalDate?): String {
+    return date?.format(DateTimeFormatter.ofPattern("dd/MM/yy")) ?: "--/--/--"
+  }
+
+  /**
+   * Retrieves the hours and minutes from a [Date] object and returns a String. I null returns --:--
```

Comment:
Typo in the comment. It should be "If null returns ..."

Reply:
fixed